scripts/publish-to-npm.js

```diff
@@ -23,20 +23,22 @@ const main = () => {
 
     console.log('--- Given the current branch:', branchName)
 
-    const matched = branchName.match(RELEASE_ONE_PACKAGE)
-    const packageName = matched && matched[1]
+    const isNightly = branchName === 'nightly-releases'
```

Comment:
Need to manage npm tags to be added as `nighlty-next` or `next` depending on whether the current branch is publishing a nightly build